# required so LD_LIBRARY_PATH is not overwritten during build. 
# pull request sent to scipion developers: https://github.com/I2PC/scipion/pull/363

diff -ru scipion.orig/install/funcs.py scipion/install/funcs.py
--- scipion.orig/install/funcs.py	2016-02-22 14:44:24.000000000 +0100
+++ scipion/install/funcs.py	2016-03-09 11:40:53.522216388 +0100
@@ -476,7 +476,7 @@
 
         environ = {
             'PYTHONHOME': prefix,
-            'LD_LIBRARY_PATH': '%s/lib' % prefix,
+            'LD_LIBRARY_PATH': '%s/lib:%s' % (prefix, os.environ.get('LD_LIBRARY_PATH')),
             'PATH': '%s/bin:%s' % (prefix, os.environ['PATH']),
             'CPPFLAGS': '-I%s/include' % prefix,
             'LDFLAGS': '-L%s/lib %s' % (prefix, os.environ.get('LDFLAGS', ''))}

